runtime/plugin/tohtml.lua

```diff
@@ -0,0 +1,11 @@
+if vim.g.loaded_2html_plugin ~= nil then
+  return
+end
+vim.g.loaded_2html_plugin = true
+
+vim.api.nvim_create_user_command('TOhtml', function(args)
+  local outfile = args.args ~= '' and args.args or vim.fn.tempname() .. '.html'
+  local html = require('tohtml').tohtml()
+  vim.fn.writefile(html, outfile)
+  vim.cmd.split(outfile)
```

Comment:
I think this should be `vim.cmd.new(outfile)`.

Reply:
I haven't looked into the source code but in the documentation `:split {file}` and `:new {file}` are the same thing. (`:help :split_f`)
(Although `:split [file]` appears in `:help :split`)

Reply:
Yeah, that's not a proper fix, sorry (it worked in one test, and then I jumped the gun).